docs/specs/iota_did_method_spec.md

```diff
@@ -0,0 +1,324 @@
+
+The IOTA Identity implementation currently supports:
+- `JcsEd25519Signature2020`
+
+**Verification Method**
+
+The proof object MUST include a `verificationMethod` which references a verification method embedded in the same DID Document. 
+
+**Signature**
+
+Depending on the verification method, a set of fields are REQUIRED to provide the cryptographic proof of the DID Document. For example, the `JcsEd25519Signature2020` method has a `signatureValue` field. 
+
+Example `proof` using the `JcsEd25519Signature2020` method:
+```json
+"proof": {
+    "type": "JcsEd25519Signature2020",
+    "verificationMethod": "did:iota:GzXeqBXGCbuebiFtKo4JDNo6CmYmGbqxyh2fDVKadiBG#authentication",
+    "signatureValue": "3fLtv3KUU4T5bHNLprV3UQ2Te3bcRZ9uUYSFouEA7fmYthieV35NNLqbKUu8t2QmzYgnfp1KMzCqPzGNi3RjU822"
+}
+```
+
+### Autonomy of the Diff object
+
+The `diff` object MUST contain all the differences between the previous DID Document and the current DID Document. The differentiation is formatted as an escaped JSON object that compares the two DID Document objects to generate a difference JSON object. Exact details of how this is generated will be added later. 
+
+Example `diff` of adding an `authentication` key and changing the `updated` field:
+```json
+{
+  "diff": "{\"authentication\":[{\"index\":0,\"item\":{\"Embed\":{\"id\":\"did:iota:7EhyBxAhFXojqzrKt8Zq7QBvxNZWJJ4xj1mm2QgLmcKj#key-3\",\"key_data\":{\"PublicKeyBase58\":\"TJqJAnV387wTUfzq8BVE7iJ9LYs7xJYM4SEF86LkB8E\"}}}}],\"properties\":[{\"c:k\":\"updated\",\"c:v\":\"2021-04-15T10:31:21Z\"}]}"
+} 
+```
+
+## CRUD Operations
+
+Create, Read, Update and Delete (CRUD) operations that change the DID Documents are to be submitted to an IOTA Tangle in order to be publicly avaliable. They will either have to be a valid Int DID message or Diff DID message, submitted on the correct indexation for the identity. 
+
+### Create
+
+To generate a new DID, the method described in [generation](#generation) must be followed. A basic DID Document must be created that includes the public key used in the DID creation process as an `verificationMethod`. This DID Document must be formatted as an Integration DID message and published to an IOTA Tangle on the indexation generated out of the public key used in the DID creation process. 
+
+### Read
+
+To read the latest DID Document associated with a DID, the following process must be executed:
+* Query all the Integration DID messages from the indexation, which is the `tag` part of the DID.
+* Order the messages based on `previous_message_id` linkages. See [Determining Order](#determining-order) for more details.
+* Validate the first Integration DID message as containing a public key inside the `verificationMethod` field that, when hashed using the `BLAKE2b-256` hashing algorithm, equals the `tag` field of the DID. 
```

Comment:
bit confusing sentence.... did you mean:
message as containing a public  ->  message that contains a public